core/idl/database/database.djinni

```diff
@@ -240,10 +237,10 @@ DatabaseBackend = interface +c {
     # @return trye if query logging is enabled, false otherwise.
     isLoggingEnabled(): bool;
 
-    # Create an instance of SQLite3 database
+    # Create an instance of SQLite3 database.
     # @return DatabaseBackend object
     static getSqlite3Backend(): DatabaseBackend;
 
-    # Create an instance of PostgreSQL database
+    # Create an instance of PostgreSQL database.
```

Comment:
Ouch "Copier/Gaufrer" (my bad). It should be something like create a database backend instance from the given DatabaseEngine implementation.

Reply:
I’ll fix it! Thanks for the review 🙇